### DISCUSSION TITLE
Elevate Warnings to Errors

### DISCUSSION THREAD

**Discussion by chrpai** [wixtoolset/issues#8744] [Questions]
Is there a way at build time to elevate WIX8601 to an error instead of a warning?

For example here are two very bad behaviors that result in a "warning" and a very wrong installer being built.

Include a directory that doesn't exist  ... WIX8601.   All the files are missing.
Exclude a directory that doesn't exist ... WIX8601.   Not only are the files from the missing directory missing (we were excluding them anyways) but  *ALL* files  mentioned in the Files@Include attribute are also missing.

**Comment by bevanweiss** [comment]
I believe this can be done as an MSBuild property
https://github.com/dotnet/msbuild/pull/5774

```xml
<PropertyGroup>
  <WarningsAsErrors>WIX8601</WarningsAsErrors>
</PropertyGroup>
```
